lib/sidekiq/prometheus/exporter.rb

```diff
@@ -12,7 +12,7 @@ module Exporter
       NOT_FOUND_TEXT = 'Not Found'.freeze
       MOUNT_PATH = '/metrics'.freeze
       # rubocop:disable Style/MutableConstant
-      HEADERS = {'Content-Type' => 'text/plain; version=0.0.4', 'Cache-Control' => 'no-cache'}
+      HEADERS = {Rack::CONTENT_TYPE => 'text/plain; version=0.0.4', Rack::CACHE_CONTROL => 'no-cache'}
```

Comment:
Just to protect ourselves from non-existing constants, let's add Rack dependency like this

```ruby
spec.add_dependency 'rack', '>= 1.6.0'
```